spec/unit/spark_api/models/news_feed_meta_spec.rb

```diff
@@ -30,6 +37,13 @@
         "YearBuilt", "BuildingAreaTotal", "PropertySubType", "SubdivisionName", "MLSAreaMinor"]
       expect(news_feed_meta.core_fields).to eq field_array    
     end
+
+    it "it shouldn't modify the original fields and should return the same results when run twice" do
```

Comment:
No, you are right. I also try to avoid using "should" as the first word of every test description too.